src/layers/mod.rs

```diff
@@ -17,6 +17,10 @@ pub mod sctp;
 
 pub mod m3ua;
 
+<<<<<<< HEAD
```

Comment:
This will fail. Please try to run locally `cargo build` and `cargo test` before pushing, so that trivial errors can be caught.

Reply:
My apologies, I have made the changes now